user/rime-data/template.py

```diff
@@ -0,0 +1,50 @@
+pkgname = "rime-data"
+pkgver = "0.20250123"
+pkgrel = 0
+_plum_commit = "4c28f11f451facef809b380502874a48ba964ddb"  # r126.20240417
+_subrepo_commits = {
+    "bopomofo": "c7618f4f5728e1634417e9d02ea50d82b71956ab",  # r59.20210131
+    "cangjie": "0ac8452eeb4abbcd8dd1f9e7314012310743285f",  # r73.20240325
+    "essay": "4dfd95ab166d542a10647753d17ce83126f70ff6",  # r265.20250121
+    "luna-pinyin": "d2107f46dbffbe069b10072925be3c18da40fe77",  # r290.20250123
+    "prelude": "3803f09458072e03b9ed396692ce7e1d35c88c95",  # r154.20240519
+    "stroke": "7c9874c6b2e0b94947653e9a7de6f99623ff27e4",  # r56.20250118
+    "terra-pinyin": "333ec4128fa1f93924a0707da3c623ccd92a73f0",  # r270.20241225
+}
+build_wrksrc = f"plum-{_plum_commit}"
```

Comment:
after changing the sources stuff that you should also be able to get rid of this